index.html

```diff
@@ -11,15 +11,15 @@
     <div class="jumbotron">
       <div class="container">
         <img src="/assets/images/trucol-logo.svg" alt="TruCol logo">
-        <p><i>TruCol</i> is a decentralised collaboration protocol, which promotes <em>inclusive</em>, <em>fair</em> and <em>accessible</em> work, by
-          enabling developers to participate in the market regardless of their geosocial circumstances, gender or ethnicity.</p>
+        <p><i>TruCol</i> is a decentralised collaboration protocol for test-driven development, which promotes <em>inclusive</em>, <em>fair</em> and <em>accessible</em> work, by
+          enabling developers to participate in the market regardless of their circumstances.</p>
```

Comment:
I'm not sure whether I remember correctly but the the ``` regardless of their circumstances ``` part was changed at @a-t-0 request. I think I would actually changed the whole sentence to just 
```TruCol is a decentralised collaboration protocol for test-driven development, which promotes inclusive,fair and accessible work ```. I think the words inclusive, fair and accessible already imply the stuff mentioned in the second part of the sentence( the part I have removed in my suggestion).

Reply:
Thanks for your vigilance both of you! I think the solution by @Avuerro is elegant!